linode/manager
# refactor: [M3-6522] - Tile and ShowMoreExpansion v7 Stories

## Description 📝
Small PR to migrate mdx to tsx stories for both Tile and ShowMoreExpansion components

## Changes  🔄
- Migrate to v7 stories
- Add test coverage
- Move styles to own file as necessary

## Preview 📷
![Screenshot 2023-11-27 at 12 28 31](https://github.com/linode/manager/assets/130582365/c02a7399-d381-48ac-baf2-f89b49e8b0c9)

![Screenshot 2023-11-27 at 12 29 03](https://github.com/linode/manager/assets/130582365/7e02d010-235f-410e-b2d4-3ee596c5e410)

## How to test 🧪

### Verification steps 
- run `yarn storybook`
- confirm stories, prop table and behavior
- run `yarn up` and make sure there isn't any regressions in either component

## As an Author I have considered 🤔

*Check all that apply*
